### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: doc/DESCRIPTION.md [hunk 1]
```diff
@@ -1 +1 @@
-Focalboard is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view.
\ No newline at end of file
+Focalboard is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view. Focalboard comes in two editions:
```
Comment (tituspijean) [inline doc/DESCRIPTION.md:1]: Do not let us hanging, which ones? 😛

**Review by tituspijean — APPROVED** [review]
LGTM, and upgraded at home without issue. 🚀

File: doc/DESCRIPTION.md (hunk 1 above)
Comment (ericgaspar) [reply doc/DESCRIPTION.md:1]: Well, it is like in the comics, you need to turn the page 😅

**Pull request merged**